From 7a24ba0fd578c6c164f5f34d14caa2a4ea4cfb7e Mon Sep 17 00:00:00 2001
From: Martin Hauke <mardnh@gmx.de>
Date: Mon, 14 Dec 2020 21:21:10 +0100
Subject: [PATCH] Fix compiler warning

ImageEffectFactory.cpp:34:1: error: control reaches end of non-void function [-Werror=return-type]
---
 src/annotations/core/imageEffects/ImageEffectFactory.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/annotations/core/imageEffects/ImageEffectFactory.cpp b/src/annotations/core/imageEffects/ImageEffectFactory.cpp
index ecfc530..1f58f95 100644
--- a/src/annotations/core/imageEffects/ImageEffectFactory.cpp
+++ b/src/annotations/core/imageEffects/ImageEffectFactory.cpp
@@ -30,5 +30,6 @@ QGraphicsEffect *kImageAnnotator::ImageEffectFactory::create(kImageAnnotator::Im
 			return new BorderImageEffect;
 		case ImageEffects::Grayscale:
 			return new GrayscaleImageEffect;
+                default: __builtin_unreachable();
 	}
 }
-- 
2.29.2

